.env.example

```diff
@@ -0,0 +1,5 @@
+## API Url
+REACT_APP_BACKEND_URL=https://localhost:44309
```

Comment:
I suppose there are two ports, aren't there?  

1. Run in IDE (44309)
2. Run in cmd (5001)

Reply:
Yes, you are right, the port is different and you can find the definition inside 👇 
`src\WebApi\WebApi\Properties\launchSettings.json`

Since it's only a .env.example this won't affect your local setup. It works as a guide for the creation of a new .env file for first start.